### PR TITLE
Add function to enable laser game object

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: Assets/Scripts/Player.cs [hunk 1]
```diff
@@ -10,10 +10,18 @@ public class Player : MonoBehaviour
...
     private float lastTimeFired;
+    public float laserLifeTime;
     public Transform fireFrom;
     public GameObject fireBullet;
+    public GameObject laser;
     public AudioSource fireSound;
 
+    void Start ()
+    {
+        enableLaser ();
```
Comment (AmirSavand) [inline Assets/Scripts/Player.cs]: Remove debug line

File: Assets/Scripts/Player.cs [hunk 2]
```diff
@@ -10,10 +10,18 @@ public class Player : MonoBehaviour
...
+        Invoke ("disableLaser", laserLifeTime);
```
Comment (AmirSavand) [inline Assets/Scripts/Player.cs]: This should be in `enableLaser` function.

Comment (ArsalanSavand) [reply Assets/Scripts/Player.cs]: Done.

File: Assets/Scripts/Player.cs (hunk 1 above)
Comment (ArsalanSavand) [reply Assets/Scripts/Player.cs]: Done.

File: Assets/Scripts/Player.cs [hunk 3]
```diff
@@ -10,8 +10,10 @@ public class Player : MonoBehaviour
     public float fireLifetime = 1;
     public float fireRate = 1;
     private float lastTimeFired;
+    public float laserLifeTime;
```
Comment (AmirSavand) [inline Assets/Scripts/Player.cs]: Rename to `laserDuration`.

**Pull request merged**